core/src/util/usLDAPFilter.cpp

```diff
@@ -0,0 +1,131 @@
+/*=============================================================================
```

Comment:
as we discussed offline, see if you can make git recognize this as a move of a file instead of a whole new file. If that is impossible, please summarize what areas have changed for code review. Thanks.